packages/kbn-securitysolution-io-ts-list-types/src/request/import_exception_list_schema/index.test.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { left } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { exactCheck, foldLeftRight, getPaths } from '@kbn/securitysolution-io-ts-utils';
+
+import { importExceptionsListSchema, ImportExceptionsListSchema } from '.';
+import {
+  getImportExceptionsListSchemaMock,
+  getImportExceptionsListSchemaDecodedMock,
+} from './index.mock';
+
+describe('import_list_item_schema', () => {
+  test('it should validate a typical lists request', () => {
+    const payload = getImportExceptionsListSchemaMock();
+    const decoded = importExceptionsListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(getImportExceptionsListSchemaDecodedMock());
+  });
+
+  test('it should NOT accept an undefined for "list_id"', () => {
+    const payload = getImportExceptionsListSchemaMock();
+    // @ts-expect-error
```

Comment:
You should avoid the `ts-expect-error` using regular TypeScript as this is just debt.

You can use a Partial + ReturnType like so:
```ts
    const payload: Partial<ReturnType<typeof getImportExceptionsListSchemaMock>> =
      getImportExceptionsListSchemaMock();
```

Like always, we want to not use them unless we have absolutely no alternative.  In this case it's just you casting to a partial and everything should work out.